### ISSUE TITLE
Hover/select thumbnail in case of unclusterable POI

### ISSUE THREAD

**Issue by josefspr** [daat-hamakom/maps#99]
Now it just opens one of the thumbnails, I think that is ok and not necessary to create a special thumbnail, but what would clarify is that it would look like there are other thumbnails beneath it, you 
know what I mean ?

**Comment by mushon** [comment]
We can make undeclusterable clusters slightly bigger (8x8px) and give them full opacity unlike the 50% transparent single markers. @yuvadm can we try this? Do you need anything from me to make this happen?

**Comment by mushon** [comment]
- we need to design the hover for it @amitay7 can you look into it?

**Comment by yuvadm** [comment]
@mushon @amitay7 yes I need new assets for this. THe opacity I can do myself, but the size isn't something I can do.
